docs/roadmap.markdown

```diff
@@ -4,9 +4,122 @@ title: "Roadmap"
+   * Initial implementation of
+     * Deployment Markers
+     * Tagging
+   <br/>
+
+* **[New Relic AWS CloudFormation Integration](https://github.com/newrelic/cloudformation-partner-integration)** - The AWS CloudFormation Integration enables developers using CloudFormation to easily provision a subset of New Relic resources.
+
+   * Build on the NR Client
+   * Initial implementation of
+     * Basic NRQL Alert configuration
+   <br/>
+
+
+## Milestones
+
+Ongoing work will be tracked in the [Developer Toolkit](https://github.com/orgs/newrelic/projects/6) Project, which will link to each individual tool that is in flight.  Here the latest summary for planned work by the New Relic team:
+
+### 2019 Winter
+
+| State | Tool | Milestone | Rationale |
+| ----- | ---- | --------- | --------- |
+| Completed | New Relic Terraform Provider | (Tracked Internally)| The current Terraform Provider for New Relic was created by the community, and suffering from lack of attention.  DTK will integrate into the ecosystem, and create a clean base to work from. |
+| **Current** | New Relic Terraform Provider | [Feature Parity with v2 APIs](https://github.com/terraform-providers/terraform-provider-newrelic/projects/1?card_filter_query=milestone%3A%22feature+parity+with+v2+apis%22) | Enhance the Terraform provider to include all capabilities of the REST v2 API and NerdGraph data access, including configuring Cloud Integrations and Tagging |
+| Next | New Relic Client | [New Relic Client Consolidation](https://github.com/newrelic/newrelic-client-go/projects/1?card_filter_query=milestone%3A%22new+relic+client+consolidation%22) | Currently there is not a consistent, easy to use experience for developing against the New Relic APIs.  As the Developer Toolkit team continues to create new tools for users, having a consolidated library.  This MMF strives to create a unified client library for consumption by the community, and specifically utilized in the New Relic Terraform Provider. |
+
+### 2020 Spring
+
+| State | Tool | Milestone | Rationale |
+| ----- | ---- | --------- | --------- |
+| Planning | New Relic Client | GraphQL Support | NerdGraph, New Relic's implementation of GraphQL is the future platform for all API requests, and is the backing solution for New Relic One. As such, the NR Client needs to implement GraphQL to support any additional data. |
+| Planning | New Relic Terraform Provider | Provider 2.0 | In order to add some newer features to the New Relic Terraform Provider, we need to implement some breaking changes.  This is a delicate area, so reving the major version and implementing the most important of those changes is warranted. |
+| Planning | New Relic CLI | Initial Support | Community members with existing workflows want to be able to leverage command line tooling for interacting with New Relic. Leveraging the New Relic Client, we can build a CLI to simplify user interactions with New Relic in scripted and interactive workflows. |
```

Comment:
Change "for interacting with New Relic" to "to interact with New Relic".  Also, I would remove New Relic from the last sentence since it's kind of redundant - just say "we can build a CLI to simplify user interactions in scripted and interactive workflows"